spec/models/product_spec.rb

```diff
@@ -0,0 +1,79 @@
+require "rails_helper"
+
+RSpec.describe Product, type: :model do
+  subject{ FactoryBot.create :product }
+  describe "validations" do
+    it { should validate_presence_of(:author) }
+    it { should validate_presence_of(:publisher) }
+
+    context "name" do
```

Comment:
context khác gì với describe và it hả e?

Reply:
- **Describe** dùng để tạo một example group, mô tả cho một thực thể(  Who?, What? )
- Còn **Context** dùng để mô tả các điều kiện (when?, if?) cho các example của thực thể 
- It thì dùng để định nghĩa một test case